source/_posts/2024-11-14-roadmap-2024h2.markdown

```diff
@@ -0,0 +1,185 @@
+
+<img src='/images/blog/2024-06-voice-chapter-7/voice-script.png' alt="Exposing voice scripts to Assist in Home Assistant Voice settings" class='no-shadow'>
+
+You can now expand LLM capabilities by [allowing them access your scripts](/blog/2024/07/03/release-20247/#expand-an-llm-using-your-scripts). This feature lets you precisely control what the LLM does in specific situations, while still benefiting from the flexibility and natural language processing that AIs excel at.
+
+#### In progress: Investigate on privacy
+
+Since our last roadmap update, we have expanded the product and design team working on the Home Assistant project, and we have hired our first ever dedicated user researcher!
+
+While we are ramping up our user research pipeline, we have started working on investigating what privacy means to our users. Of the three Open Home values, privacy, choice, and sustainability, while we have a pretty solid understanding of choice and some understanding of sustainability, the idea of privacy can benefit from more clarity. We have interviewed a handful of users on this topic recently, and we can’t wait to show you what we have found.
+
+## Home Assistant Cloud
+
+Home Assistant Cloud, powered by our commercial partner Nabu Casa brings, to the many, the hard-to-reach features of Home Assistant usually reserved for power users with expensive hardware at home, with no compromise on privacy.
+
+In recent months, the Nabu Casa team has been hard at work in bringing peace of mind to using Home Assistant. For the cloud backup feature, the Home Assistant team and the Nabu Casa team are now working closely together on a comprehensive revamp of backup integrations, to create an easy one-stop solution to a good backup strategy. We are also working to ensure users a smooth onboarding for backing up to Home Assistant Cloud, as well as other third-party backup locations.
+
+The team had to postpone some features in favor of others like [WebRTC](/blog/2024/11/06/release-202411/#low-latency-cameras-using-webrtc). They had successfully released the infrastructure needed to support WebRTC, both STUN servers for every Home Assistant user and TURN servers for every Home Assistant Cloud subscriber, enhancing real-time communication capabilities within Home Assistant. Two-factor authentication (2FA) is one of the postponed features, but it is still in the Nabu Casa pipeline.
+
+## Open Protocols
+
+We remain committed to supporting open smart home standards, including Z-Wave Zigbee, Matter, and Thread, aligning with our Open Home mission to offer choice, privacy, and sustainability through locally controlled, cloud-independent devices.
+
+In the past six months, we have made significant steps towards certifications, ensuring our open standards implementations are up to industry standards.
+
+### Z-Wave
```

Comment:
Last quarter, the focus on preparing for certification has been the biggest focus from the engineering end. Should we mention that?

Reply:
Ok! I've added a mention of that 👍